riocli/device/tools/rapyuta_logs.py

```diff
@@ -25,6 +25,11 @@
 @click.argument('device-name', type=str)
 @name_to_guid
 def rapyuta_agent_logs(device_name: str, device_guid: str) -> None:
+    """Stream rapyuta agent logs from the device.
```

Comment:
thought: Maybe we can consider updating this command to stream deviceedge's logs now.

Reply:
Yes. Let's create a task for it and implement it in another PR.